docs/pages/guides/spend-permissions/api-reference.mdx

```diff
@@ -0,0 +1,275 @@
+# Spend Permissions API Reference
+
+A more detailed reference guide for integrating with Spend Permissions.
+
+Integrating with Spend Permissions involves interacting with several APIs:
+- direct interaction with the `SpendPermissionManager` smart contract
+- interaction with the users' connected Coinbase Smart Wallet
+- optionally, using the Coinbase Wallet API to fetch information about existing permissions
+
+## `SpendPermissionManager.sol` smart contract
+
+### Structs
+
+#### `SpendPermission`
+Defines the complete parameters of a spend permission.
+
+| Field       | Type     | Description                                                                 |
+|-------------|----------|-----------------------------------------------------------------------------|
+| `account`   | `address`| Smart account this spend permission is valid for.                           |
+| `spender`   | `address`| Entity that can spend `account`'s tokens.                                   |
+| `token`     | `address`| Token address (ERC-7528 native token address or ERC-20 contract).           |
+| `allowance` | `uint160`| Maximum allowed value to spend within each `period`.                        |
+| `period`    | `uint48` | Time duration for resetting used `allowance` on a recurring basis (seconds).|
+| `start`     | `uint48` | Timestamp this spend permission is valid after (unix seconds).              |
+| `end`       | `uint48` | Timestamp this spend permission is valid until (unix seconds).              |
+| `salt`      | `uint256`| An arbitrary salt to differentiate unique spend permissions with otherwise identical data. |
+| `extraData` | `bytes`  | Arbitrary data to include in the signature.                                 |
+
+---
+
+#### `SpendPermissionBatch`
```

Comment:
would skip any documentation for batching behavior until Keys web is ready to handle it